.travis.d/definition_check.py

```diff
@@ -0,0 +1,67 @@
+import os
+import sys
+import re
+import fnmatch
+
+CURR_DIR = os.path.dirname(os.path.realpath(__file__))
+DEF_FILE_PATH = os.path.join(CURR_DIR, '..', 'docs', 'definitions.dox')
+INCLUDE_DIR = os.path.join(CURR_DIR, '..', 'include')
+
+DEFINE_RULE = re.compile(r' \*     \\defgroup ([a-zA-Z]+) [a-zA-Z ]+')
+USER_GROUP_RULE = re.compile(r' \* \\(user|kernel)group\{([a-zA-Z]+)\}')
```

Comment:
Yes, but someone doing a pull request might get frustrated trying to understand why the group is not detected even though it's there.